src/lib/connectors/shared/remote-debugging-connector.ts

```diff
@@ -73,6 +74,10 @@ export class Connector implements IConnector {
         const defaultOptions = {
             loadCompleteRetryInterval: 250,
             maxLoadWaitTime: 30000,
+            // tabUrl is a empty html site used to avoid edge diagnostics adapter to receive unexpeted onLoadEventFired
+            // and onRequestWillBeSent events from the default url opened when you create a new tab in Edge.
+            tabUrl: 'http://empty.sonarwhal.com/',
```

Comment:
Let's use `https://empty.sonarwhal.com`
I'll configure the SSL certificate once the DNS records are working.